rtt_rosparam/include/rtt_rosparam/ros_param_data_source.hpp

```diff
@@ -0,0 +1,151 @@
+/***************************************************************************
+ *        This file is part of the OROCOS toolchain ROS project
+ *
+ *                  (C) 2020, Intermodalics BVBA
+ *
```

Comment:
Please align the license header style and wording with the latest updates in #154.